src/mmrelay/meshtastic_utils.py

```
@@ -1052,6 +1055,26 @@ def _track_health_probe_request_id(
     return request_id
 
 
+def _seed_connect_time_skew(rx_time: float) -> None:
+    """Seed rxTime clock skew from a connect-time packet if not yet calibrated.
+
+    Only calibrates when the packet's rxTime is at or after RELAY_START_TIME
+    so stale backlog packets do not produce a wildly wrong skew value.
+    """
+    global _relay_rx_time_clock_skew_secs
```

Ruff (0.15.7)

[warning] Using the global statement to update `_relay_rx_time_clock_skew_secs` is discouraged

(PLW0603)